packages/web/app/views/programRegistry/ConditionSection.jsx

```diff
@@ -112,10 +112,26 @@ export const ConditionSection = ({ patientProgramRegistration, programRegistryCo
   return (
     <Container>
       <HeadingContainer>
-        <Heading5>Related conditions</Heading5>
-        <ConditionalTooltip title="Patient must be active" visible={isRemoved}>
+        <Heading5>
+          <TranslatedText
+            stringId="patientProgramRegistry.general.relatedConditions.label"
```

Comment:
I tried to re-use as many string ids as possible and prefixed the re-usable ones in the `patientProgramRegistry.general` key